code/datums/mutations/_combined.dm

```diff
@@ -17,10 +17,12 @@
 	required = "/datum/mutation/human/strong; /datum/mutation/human/radioactive"
 	result = HULK
 
+//Commented in an effort to keep the code for a potential later genetics rework maybe?
```

Comment:
Don't do this, remove the code in question, we use git for a reason

Reply:
Done!